README.md

```diff
@@ -50,7 +50,7 @@ Kitura is a web framework and web server that is created for web services writte
 
 ## Getting Started
 
-Visit [www.kitura.io](http://www.kitura.io/) for a Getting Started guide, tutorials, and API reference documentation.
+Visit [https://www.kitura.dev](http://www.kitura.dev/) for a Getting Started guide, tutorials, and API reference documentation.
```

Comment:
Would you mind setting renaming http to https in the text as well?

Reply:
Done. And I've modified 2 more links.
There is also a link pointing to https://www.kitura.io/packages.html#al, but I don't know where it should point now.